include/rocksdb/options.h

```diff
@@ -1962,4 +1962,8 @@ struct LiveFilesStorageInfoOptions {
 };
 #endif  // !ROCKSDB_LITE
 
+struct MergeInstanceOptions {
+  bool merge_memtable = false;
```

Comment:
It has an additional requirement that `write_buffer_manager` is shared so defaulting is bad.

Reply:
How about naming it `merge_memtable_if_possible`? Just an example name to show the default behavior can be improved.

Reply:
I prefer to keep it because `merge_memtable` is very relevant to the freshness of target DB and user should be very aware of whether it is actually merged.  I'll later change the behavior to abort if `merge_memtable && write_buffer_manager isn't shared`.